topo/bond/forch.yaml

```diff
@@ -9,6 +9,14 @@ site:
       port: 9020
 orchestration:
   static_device_placement: placement.yaml
+proxy_server:
```

Comment:
also wouuld expect an update/change to the integration testing. It should be nearly transparent, just need to update the URL somewhere for what it's pointing at to test a raw varz access (which I believe is there somewhere).  Should be easy to check both proxied and raw access.

Reply:
Yup. Currently added to test_sol. Will replace other tests.